File: app/src/main/java/com/wolfteam20/schedulemobile/data/models/PeriodoAcademicoDTO.java
package com.wolfteam20.schedulemobile.data.models;

import java.util.Date;

/**
 * Created by Efrain.Bastidas on 1/8/2018.
 */

public class PeriodoAcademicoDTO {
    private String nombrePeriodo;
    private int idPeriodo;
    private boolean status;
    private Date fechaCreacion;

    public String getNombrePeriodo() {
        return nombrePeriodo;
    }

    public void setNombrePeriodo(String nombrePeriodo) {
        this.nombrePeriodo = nombrePeriodo;
    }

    public int getIdPeriodo() {
        return idPeriodo;
    }

    public void setIdPeriodo(int idPeriodo) {
        this.idPeriodo = idPeriodo;
    }

    public boolean isStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public Date getFechaCreacion() {
        return fechaCreacion;
    }

    public void setFechaCreacion(Date fechaCreacion) {
        this.fechaCreacion = fechaCreacion;
    }
}

File: app/src/main/java/com/wolfteam20/schedulemobile/data/network/models/PeriodoAcademicoDTO.kt
package com.wolfteam20.schedulemobile.data.network.models

import android.annotation.SuppressLint
import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import java.util.*

/**
 * Created by Efrain Bastidas on 2/4/2018.
 */

@SuppressLint("ParcelCreator")
@Parcelize
data class PeriodoAcademicoDTO(
    var idPeriodo: Long,
    var nombrePeriodo: String,
    var status: Boolean,
    var fechaCreacion: Date
) : Parcelable